README.md

```diff
@@ -131,6 +131,14 @@ customSearchEngines:
     blank: https://www.npmjs.com
 ```
 
+In case of making multilingual searching correctly, you may need to tweak [uFuzzy](https://github.com/leeoniya/uFuzzy) options via option `ufuzzyOptions`, for example:
+
+```yaml
+# make CJK chars can be searched
+ufuzzyOptions:
+  - interSplit: (p{Unified_Ideograph=yes})+
+```
```

Comment:
Yes, object format is better for code and user side specification.

I placing the array format since I found the yaml option page saving
will automatically converts my inputs of object list to array in that
time that I haven't found a way for making the option setting using
yaml object format which caused me to follow the format type as what
the builtin `customSearchEngines` does.

Thanks for fix this, great!